tests/third_party/cupy/manipulation_tests/test_rearrange.py

```diff
@@ -0,0 +1,76 @@
+import unittest
+
+import numpy
+import pytest
+
+import dpnp
```

Comment:
It would be easier just to make `import dpnp as cupy`, instead of replacing cupy with dpnp everywhere in file